src/schema/entities.yaml

```diff
@@ -118,6 +118,24 @@ recording:
     In such case use different labels.
     For example: `_recording-contrast`, `_recording-saturation`.
   format: label
+part:
+  name: Part
+  entity: part
+  description: |
+    This entity is used to indicate which component of the complex
+    representation of the MRI signal is represented in voxel data.
+    The `part-<label>` key/value pair is associated with the DICOM tag
+    `0008,9208`.
+    Allowed label values for this entity are `phase`, `mag`, `real` and `imag`,
+    which are typically used in `mag/phase` or `real/imag` pairs.
```

Comment:
see comment above on that I find the `mag/phase` suggestive of being some kind of syntax we support (but we don't)